bucket/windows-terminal-preview.json

```diff
@@ -52,12 +52,32 @@
             "Windows Terminal Preview"
         ]
     ],
+    "persist": [
+        ".portable",
+        "settings"
+    ],
+    "pre_uninstall": "if ($cmd -eq 'uninstall') { reg import \"$dir\\uninstall-context.reg\" }",
     "checkver": {
         "url": "https://api.github.com/repos/microsoft/terminal/releases",
-        "jsonpath": "$.[?(@.prerelease == true)]",
-        "regex": "\"tag_name\": \"v([\\d.]+)\"[\\s\\S]+?Win10_(?<version>[\\d.]+)_8wekyb3d8bbwe"
+        "jsonpath": "$[?(@.prerelease == true)].tag_name",
+        "regex": "v([\\d.]+)"
     },
     "autoupdate": {
-        "url": "https://github.com/microsoft/terminal/releases/download/v$version/Microsoft.WindowsTerminalPreview_Win10_$matchVersion_8wekyb3d8bbwe.msixbundle#/dl.7z"
+        "architecture": {
+            "64bit": {
+                "url": "https://github.com/microsoft/terminal/releases/download/v$version/Microsoft.WindowsTerminalPreview_$version_x64.zip"
+            },
+            "32bit": {
+                "url": "https://github.com/microsoft/terminal/releases/download/v$version/Microsoft.WindowsTerminalPreview_$version_x86.zip"
+            },
+            "arm64": {
+                "url": "https://github.com/microsoft/terminal/releases/download/v$version/Microsoft.WindowsTerminalPreview_$version_arm64.zip"
+            }
+        },
+        "hash": {
+            "url": "https://github.com/microsoft/terminal/releases/tag/v$version",
+            "regex": "(?s)$basename.*?$sha256"
```

Comment:
The hash check won't work for [1.18.1421.0](https://github.com/microsoft/terminal/releases/tag/v1.18.1421.0) because the asset hashes weren't included in the release notes for some reason. Updating to [1.18.1462.0](https://github.com/microsoft/terminal/releases/tag/v1.18.1462.0) solves this problem as well as fixing a bunch of issues.